mrp_partial_production/demo/stock_demo.xml

```diff
@@ -26,10 +26,13 @@
             <field name="location_id" ref="stock.stock_location_stock"/>
         </record>
 
+        <function model="stock.inventory" name="prepare_inventory">
+            <function eval="[[('id', '=', ref('stock_inventory_pick_0'))]]" model="stock.inventory" name="search"/>
+        </function>
         <function model="stock.inventory" name="action_done">
             <function eval="[[('id', '=', ref('stock_inventory_pick_0'))]]" model="stock.inventory" name="search"/>
```

Comment:
same here

Reply:
Change in [eabee15](https://github.com/Vauxoo/addons-vauxoo/pull/1227/commits/eabee156f7666fc2b1bf0f43b06e19a1659bf5bf)